src/sentry/snuba/events.py

```diff
@@ -463,11 +463,10 @@ class Columns(Enum):
         alias="transaction.status",
     )
     TRANSACTION_SOURCE = Column(
-        group_name=None,
-        event_name=None,
+        group_name="events.transaction",
+        event_name="transaction_source",
         transaction_name="transaction_source",
-        # Only available in transactions, pretend like its a tag so we dont error and just null instead
-        discover_name="tags[transaction_source]",
+        discover_name="transaction_source",
```

Comment:
@wmak / @k-fish FYI. I've changed since we need it in discover query. lmk wdyt?

Reply:
AFAIK transaction_source isn't in the discover dataset, and only in the transactions dataset?

Reply:
fixed.